modules/programs/vim.nix

```
@@ -65,6 +65,18 @@ in {
     programs.vim = {
       enable = mkEnableOption "Vim";
 
+      source_package = mkOption {
```

Options should as much as possible use camel cased names.
```suggestion
      sourcePackage = mkOption {
```

---

Right, will change that.

---

Changed.